motionplan/solverFrame.go

```diff
@@ -244,6 +245,16 @@ func (sf *solverFrame) DoF() []frame.Limit {
 	return limits
 }
 
+func (sf *solverFrame) movingFrame(name string) bool {
+	for _, movingName := range sf.movingFrames {
+		// TODO(rb): iterating over the list and doing string compares is bad - change after you get this working
```

Comment:
Change incoming for this?

Reply:
Yeah.  I forgot I still wanted to do this.  Will come up with something